src/console_cmds.cpp

```diff
@@ -1129,6 +1129,52 @@ DEF_CONSOLE_CMD(ConNewGame)
 	return true;
 }
 
+DEF_CONSOLE_CMD(ConNewHeightMapGame)
+{
+	if (argc == 0) {
+		IConsolePrint(CC_HELP, "Load a game by name or index. Usage: 'newheightmapgame [<file | number> [seed]]'");
```

Comment:
Do we have documentation anywhere for what to put as the seed? I'm not familiar with how we use seeds, but my lack of knowledge might not mean anything. 🙂